pkg/odo/cli/preference/set.go

```diff
@@ -111,3 +104,23 @@ func NewCmdSet(name, fullName string) *cobra.Command {
 	preferenceSetCmd.Flags().BoolVarP(&o.configForceFlag, "force", "f", false, "Don't ask for confirmation, set the preference directly")
 	return preferenceSetCmd
 }
+
+func formatPreferenceExamples(fullName, templater string) string {
```

Comment:
I am not very happy about this function placement and perhaps we could write this function in a better way, so please feel free to suggest that :)